lib/core/spec/shared/requests/example_groups.rb

```diff
@@ -74,6 +74,23 @@ def mock_authentication
   end
 end
 
+RSpec.shared_context 'cognito_user' do
```

Comment:
I think this is a much needed thing moving forward. 👍

Reply:
One small thing though. please keep the context string consistent:
"cognito user" instead of "cognito_user"